core/kazoo_amqp/src/kz_amqp_sup.erl

```diff
@@ -83,7 +84,8 @@ pools() ->
 pool_name() ->
     case get('$amqp_pool') of
         'undefined' -> ?POOL_NAME;
-        Name -> Name
+        Name -> lager:warning("using pool with name ~s", [Name]),
```

Comment:
Is this really a warning-level log line?

Reply:
fixing